config/configgrpc/configgrpc.go

```diff
@@ -253,9 +254,11 @@ func (gcs *ClientConfig) toDialOptions(host component.Host, settings component.T
 
 	otelOpts := []otelgrpc.Option{
 		otelgrpc.WithTracerProvider(settings.TracerProvider),
-		otelgrpc.WithMeterProvider(settings.MeterProvider),
 		otelgrpc.WithPropagators(otel.GetTextMapPropagator()),
 	}
+	if settings.MetricsLevel >= configtelemetry.LevelDetailed {
+		otelOpts = append(otelOpts, otelgrpc.WithMeterProvider(settings.MeterProvider))
```

Comment:
Will only adding the `WithMeterProvider` in this condition break users who are emitting metrics with OTel on lower metric levels?

Reply:
Sorry I'm not sure I understand the question

Reply:
I misunderstood that `otelgrpc.WithMeterProvider` is whats configuring the grpc instrumentation, you've done this right.